plugins/versionlock.py

```diff
@@ -192,6 +192,8 @@ def run(self):
 def _read_locklist():
     locklist = []
     try:
+        if not locklist_fn:
+            raise dnf.exceptions.Error(NO_LOCKLIST)
```

Comment:
These lines just check if the filename is valid. If you are not checking the file name you can try to open a `NaN` which will reault in opening a input stream.  
> The problem with this is that when I have versionlock installed and I delete the locklist option from versionlock.conf, then I get an error even when I run dnf upgrade. Which I don't think is desirable

This is actually expected

Reply:
>Also, we don't check if locklist_fn is set in the _write_locklist (running dnf versionlock add libdnf prints that it adds libdnf, but it actually doesn't have any effect), so if you could add it, too, that would be great.

I will add the check inside the `_write_locklist` as it is in `_read_locklist`